common-extensions/src/main/java/com/twocoders/util/commonextensions/IntExtensions.kt

```diff
@@ -0,0 +1,8 @@
+package com.twocoders.util.commonextensions
+
+const val FIRST = 0
```

Comment:
neviem ci by som taketo hodnoty definoval pre vsetkych rovnako. Pride mi, ze to si kazdy projekt moze nastavit ako chce a ci niekto bude potrebovat `NO_MARGIN` alebo `NO_PADDING` int? Skor vacsinou nie, ci?

Reply:
I toto dám klidně pryč, jen víš, toto defakto doplňuje chybějící konstanty v Androidu, zatím alespoň nějaké. Např. náš oblíbený `NO_ID`, i ty v tom `DynamicText-u` jsi si vypůjčil `View.NO_ID` a porovnáváš to s resourcem, ale jinak to v té třídě nedává smysl, protože to nijak s `View` nesouvisí, jen jsi jinou konstantu neměl k dispozici a psát tam na x míst -1 také není Bůh ví co :-D Kdyby byly tyto konstanty globální a všeobecně zadefinované a zaužívané, tak bychom to tu teď nemuseli vůbec řešit :)

Reply:
Mas pravdu, niektore tam fakt mali byt (respektive uz su ale len nedavno). Stale ale tieto tu nedavaju az tak zmysel. Navyse ak NO_ID ma byt pre resources lebo tam to je to `0` a nie `-1`.

Navrhujem ked tak, zadefinovat resource NO_ID, co pre 29+ budeme normalne delegovat na [Resources.ID_NULL](https://developer.android.com/reference/kotlin/android/content/res/Resources#id_null) a pre nizsie bude vracat `0`. (Respekive priamo nadratujeme na `ResourcesCompat` lebo sak to je spatne kompatibilne.)

Toto by malo byt aj pre dynamicText, lebo tam to NO_ID fakt nedava zmysel (je sice pouzite iba interne, ale aj tak). Malo by tam byt Resources.ID_NULL lebo sa tam riesia string resources a pokryje to rovno aj `0` vstup. A ak to nechceme pokryvat, malo by to byt nejake private ID, ktore je interne pre tu triedu...